src/expr.js

```diff
@@ -191,7 +191,7 @@ class Unary extends Expr {
         this.right = right;
     }
 
-    accept(visitor) {
+    adicionar(visitor) {
```

Comment:
Deveria ser `aceitar`, não?

Reply:
@leonelsanchesdasilva No primeiro momento eu tinha feito `aceitar` mas achei que adicionar faria mais sentido.
Acha que faz mais sentido se for `aceitar` ?

Reply:
@samuelrvg Sim, acho.

Reply:
Resolvido no commit a22bf1